pkgs/development/python-modules/drawille/default.nix

```
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "drawille";
+  version = "0.1.0";
+
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-t4nS8TWbEGKHibIbLfZZycPQxTiEzuJ7DYsa6Twi+8s=";
+  };
+
+  doCheck = false;
```

```suggestion
  doCheck = false; # pypi package has no tests, git has no tags
```

A short explanation can go a long way.